docs/release-notes.rst

```diff
@@ -19,4 +19,179 @@
+       | - Added support for NVIDIA BlueField-3 NIC mode.
+       | - Added High-Availability and Leader election support for NV-IPAM.
+       | - Added systemd mode support for SR-IOV Network Operator and MOFED container to optimize cluster/node startup time.
+   * - 23.7.0
+     - | - Added support for OpenShift Container Platform 4.13.
+       | - Added support for RHEL 9.1 and 9.2 with CRI-O container runtime (Beta).
+       | - Added support for NodeFeatureApi in Node Feature Discovery.
+   * - 23.5.0
+     - | - Added support for NVIDIA IPAM Plugin deployment.
+       | - Added support for CRDs upgrade during NVIDIA Network Operator installation or upgrade.
+   * - 23.4.0
+     - | - Added support for Kubernetes >= 1.21 and <=1.27.
+       | - Added support for NicClusterPolicy update and removal.
+       | - Added support for OpenShift Container Platform 4.11 and 4.12.
+   * - 23.4.0
+     - | - Added a calendar versioning schema for Network Operator releases to better align with the NVIDIA GPU Operator.
+       | - Added support for the following operating systems and Kubernetes environments:
+       |     - RHEL 8.4 and 8.6 with CRI-O container runtime
+       |     - Kubernetes >= 1.21 and <=1.26
+       | - Added PKey configuration for IB networks with IB-Kubernetes.
+       | - Added the ability to gracefully terminate the OFED container on DGX systems running Red Hat OpenShift.
+   * - 1.4.0
+     - | - Added support for Kubernetes >= 1.21 and <=1.25.
+       | - Added support for Ubuntu 22.04.
+       | - Added support for OpenShift Container Platform 4.11 including DGX platform.
+       | - Added Beta support for PKey configuration for IB networks with IB-Kubernetes.
+   * - 1.3.0
+     - | - Added support for Kubernetes >= 1.17 and <=1.24.
+       | - Added the option to use a single namespace to deploy Network Operator components.
+       | - Added support for automatic MLNX OFED driver upgrade.
+       | - Added support for IPoIB CNI.
+       | - Added support for Air Gap deployment.
+   * - 1.2.0
+     - | - Added support for OpenShift Container Platform 4.10.
+       | - Added extended selectors support for SR-IOV Device Plugin resources with Helm chart.
+       | - Added Whereabouts IP reconciler support.
+       | - Added BlueField2 NICs support for SR-IOV operator.
+   * - 1.1.0
+     - | - Added support for OpenShift Container Platform 4.9.
+       | - Added support for Network Operator upgrade from v1.0.0.
+       | - Added support for Kubernetes POD Security Policy.
+       | - Added support for Kubernetes >= 1.17 and <=1.22.
+       | - Added the ability to propagate nodeAffinity property from the NicClusterPolicy to Network Operator dependencies.
+   * - 1.0.0
+     - | - Added Node Feature Discovery that can be used to mark nodes with NVIDIA SR-IOV NICs.
+       | - Added support for different networking models:
+       |     - Macvlan Network
+       |     - HostDevice Network
+       |     - SR-IOV Network
```

Comment:
(nothing to address) Interesting, this nesting doesn't seem to work on github preview but works on the output html